dashboard/test/factories.rb

```diff
@@ -1,4 +1,7 @@
 FactoryGirl.define do
+  factory :coteacher do
+  end
```

Comment:
This could be removed (if not used). Alternately, I'd populate the `coteacher` with a new `user` and a new `section`.